keras_cv/layers/preprocessing3d/global_rotation.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d import base_augmentation_layer_3d
+from keras_cv.ops.point_cloud import coordinate_transform
+from keras_cv.ops.point_cloud import wrap_angle_radians
+
+POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
+BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
+
+
+class GlobalRandomRotation(base_augmentation_layer_3d.BaseAugmentationLayer3D):
```

Comment:
nit: by convention, I think this should be in global_random_rotation.py

Reply:
Done.